tests/python/unittest/test_library_loading.py

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This test checks if dynamic loading of library into MXNet is successful
+
+import mxnet as mx
+import os
+from mxnet.test_utils import download
+
```

Comment:
Can you add a decorator to skip the test if the platform is not what you expect? This will fail in other architectures. Please check OS and platform.

Wouldn't it be more resilient to build this stub as part of the build process instead of relying on downloads, this would also solve the above.

Reply:
That is a good suggestion Pedro. As discussed offline, the whole point of this PR is to load an external library, so it makes sense to download it here.

Reply:
This test won't pass in ARM. You need to check the platform and skip this test if the binary is not for the OS / platform. If you want to download an artifact, be it. But then it needs more refinement to skip the test if the platform doesn't match your binary.

Reply:
I suggest to use something like 

```
@unittest.skipIf(check_platform(), "blah")
```

If you insist in loading a binary artifact. Otherwise build the library during the build process and check that we are not pre-built as in tools/diagnose.py  for example.

```
     commit_hash = os.path.join(mx_dir, 'COMMIT_HASH')
        if os.path.exists(commit_hash):
            with open(commit_hash, 'r') as f:
                ch = f.read().strip()
                print('Commit Hash   :', ch)
        else:
            print('Commit hash file "{}" not found. Not installed from pre-built package or built from source.'.format(commit_hash))

```
So the test runs only if we are not pre-built.

I think there are two valid options.

Reply:
Added a skip unit test decorator.